src/ServiceControl/Infrastructure/WebApi/Negotiator.cs

```diff
@@ -105,7 +105,7 @@ public static HttpResponseMessage WithPagingLinks(this HttpResponseMessage respo
                 return response;
             }
 
-            var path = request.GetRouteData().Route.RouteTemplate; //should ideally be request.RequestUri.AbsolutePath, but using route instead to keep it backwards compatible
+            var path = request.RequestUri.AbsolutePath.Substring(5); // NOTE: Strips off the /api/ for backwards compat
```

Comment:
I wonder whether we should guard this substring call with a check if the `RequestUri` actually starts with `/api/`. If we ever change the prefix, it will be hard to find this code again.